components/tag-list/tag-list.js

```diff
@@ -2,7 +2,6 @@ import '../button/button-subtle.js';
 import { css, html, LitElement } from 'lit';
 import { announce } from '../../helpers/announce.js';
 import { ArrowKeysMixin } from '../../mixins/arrow-keys-mixin.js';
-import { BACKDROP_ROLE } from '../backdrop/backdrop.js';
```

Comment:
Minor: This class was exported specifically for the fix here - if we no longer need it, we could technically make it a private constant again.  Although it may be needed in the future somewhere else 🤷